lib/httpclient.rb

```diff
@@ -651,8 +651,8 @@ def redirect_uri_callback=(redirect_uri_callback)
   # use get method.  get returns HTTP::Message as a response and you need to
   # follow HTTP redirect by yourself if you need.
   def get_content(uri, *args, &block)
-    query, header = keyword_argument(args, :query, :header)
-    success_content(follow_redirect(:get, uri, query, nil, header || {}, &block))
+    query, header, to = keyword_argument(args, :query, :header, :to)
```

Comment:
Maybe we can do better in term of API. But it seemed logical to expose this optimized code path through `get_content` only.

So: `client.get_content('/big-file.bin', to: '/tmp/big-file.bin')`